incubator/hnc/config/crd/patches/singleton-validation_unserve-v1alpha1_hierarchyconfiguration.yaml

```diff
@@ -0,0 +1,165 @@
+# Please remember to re-copy CRD from bases/ if there's a CRD change. See
+# https://github.com/kubernetes-sigs/multi-tenancy/issues/1035 for detail.
```

Comment:
Automate this by adding a script in `hack/` and calling it as a part of `make generate`?

Reply:
Good idea, will work on that!